src/lib/sensor_calibration/Accelerometer.cpp

```diff
@@ -149,11 +149,27 @@ void Accelerometer::ParametersUpdate()
 			_priority = DEFAULT_PRIORITY;
 		}
 
+		bool calibration_changed = false;
+
 		// CAL_ACCx_OFF{X,Y,Z}
-		_offset = GetCalibrationParamsVector3f(SensorString(), "OFF", _calibration_index);
+		const Vector3f offset = GetCalibrationParamsVector3f(SensorString(), "OFF", _calibration_index);
+
+		if (Vector3f(_offset - offset).norm() > 0.001f) {
```

Comment:
You could avoid the square root here:
```suggestion
		if (Vector3f(_offset - offset).norm_squared() > 0.001f * 0.001f) {
```

Reply:
Thanks, might as well.